velox/connectors/hive/SplitReader.cpp

```diff
@@ -55,7 +56,13 @@ VectorPtr newConstantFromString(
                       VELOX_USER_FAIL("{}", status.message());
                     });
     if constexpr (kind == TypeKind::TIMESTAMP) {
-      copy.toGMT(Timestamp::defaultTimezone());
+      const tz::TimeZone* timezone;
+      if (sessionTimezone.empty()) {
+        timezone = &Timestamp::defaultTimezone();
+      } else {
+        timezone = tz::locateZone(sessionTimezone);
```

Comment:
This is a functionality change, let us leave it to the second PR

Reply:
Roll back to the previous logic, Thanks.